sources/functions/navidrome

```diff
@@ -0,0 +1,35 @@
+#!/bin/bash
+
+function navidrome_download_latest() {
+    echo_progress_start "Downloading release archive"
+
+    case "$(_os_arch)" in
+        "amd64") arch='x86_64' ;;
+        "arm64") arch="arm64" ;;
+        "armhf") arch="armv6" ;;
+        *)
+            echo_error "Arch not supported"
+            exit 1
+            ;;
+    esac
+
+    latest=$(curl -sL https://api.github.com/repos/navidrome/navidrome/releases/latest | grep "Linux_$arch" | grep browser_download_url | cut -d \" -f4) || {
+        echo_error "Failed to query GitHub for latest version"
+        exit 1
+    }
```

Comment:
```suggestion
github_tag="$(git ls-remote -q -t --refs https://github.com/navidrome/navidrome.git | awk '{sub("refs/tags/", ""); print $2 }' | awk '!/^$/' | sort -rV | head -n 1)"
latest="https://github.com/navidrome/navidrome/releases/download/${github_tag}/navidrome_${github_tag#v}_Linux_${arch}.tar.gz"
```

We have discussed avoiding using the api where possible and `git ls-remote` is the preferred method. Since you are testing the url when you download it you don't need to test it here as well.